graphics/src/AssimpLoader.cc

```diff
@@ -177,16 +182,50 @@ void AssimpLoader::Implementation::RecursiveCreate(const aiScene* _scene, const
   }
 }
 
+void AssimpLoader::Implementation::RecursiveStoreBoneNames(const aiScene *_scene, const aiNode *_node)
+{
+  if (!_node)
+    return;
+
+  for (unsigned meshIdx = 0; meshIdx < _node->mNumMeshes; ++meshIdx)
+  {
+    auto assimpMeshIdx = _node->mMeshes[meshIdx];
+    auto assimpMesh = _scene->mMeshes[assimpMeshIdx];
+    for (unsigned boneIdx = 0; boneIdx < assimpMesh->mNumBones; ++boneIdx)
+    {
+      auto bone = assimpMesh->mBones[boneIdx];
+      auto boneName = std::string(bone->mName.C_Str());
```

Comment:
Nit, I added a `ToString` function since this conversion is done all the time throughout the loader

Reply:
Done.

Reply:
None really, just remove duplicated code all over the place